math/pom.xml

```diff
@@ -48,6 +48,16 @@
             <groupId>gov.nist.math</groupId>
             <artifactId>jama</artifactId>
         </dependency>
+        <dependency>
+            <groupId>org.scijava</groupId>
+            <artifactId>native-lib-loader</artifactId>
+        </dependency>
+
+        <dependency>
+            <groupId>com.powsybl</groupId>
+            <artifactId>powsybl-math-native</artifactId>
+            <version>${powsyblmathnative.version}</version>
```

Comment:
Should we declare the version in the parent pom.xml to be consistent with the rest of the project?

Reply:
You also forgot to fix distribution-core/pom.xml to remove the copy of old libmathjni.so

Reply:
> Should we declare the version in the parent pom.xml to be consistent with the rest of the project?
Indeed, it is an external dependency. Done.

Reply:
> You also forgot to fix distribution-core/pom.xml to remove the copy of old libmathjni.so

Done